doc/connectivity/bluetooth/api/mesh/shell.rst

```diff
@@ -127,114 +130,122 @@ General configuration
 
 	* ``AppIdx``: If present, sets the new application index. If omitted, the current application index is printed.
 
+Low Power Node
+==============
 
-``mesh net-send <hex string>``
-------------------------------
+``mesh lpn set <value: off, on>``
+---------------------------------
 
-	Send a raw mesh message with the current destination address, network and application index. The message opcode must be encoded manually.
+	Enable or disable Low Power operation. Once enabled, the device will turn off its radio and start polling for friend nodes.
 
-	* ``hex string`` Raw hexadecimal representation of the message to send.
+	* ``value``: Sets whether Low Power operation is enabled.
 
+``mesh lpn poll``
+-----------------
+
+	Perform a poll to the friend node, to receive any pending messages. Only available when LPN is enabled.
 
 Testing
 =======
 
-``mesh iv-update``
-------------------
+``mesh test net-send <Hex string>``
+-----------------------------------
+
+	Send a raw mesh message with the current destination address, network and application index. The message opcode must be encoded manually.
+
+	* ``hex string`` Raw hexadecimal representation of the message to send.
+
+``mesh test iv-update``
+-----------------------
 
 	Force an IV update.
 
 
-``mesh iv-update-test <value: off, on>``
-----------------------------------------
+``mesh test iv-update-test <value: off, on>``
+---------------------------------------------
 
 	Set the IV update test mode. In test mode, the IV update timing requirements are bypassed.
 
 	* ``value``: Enable or disable the IV update test mode.
 
 
-``mesh rpl-clear``
-------------------
+``mesh test rpl-clear``
+-----------------------
 
 	Clear the replay protection list, forcing the node to forget all received messages.
 
 .. warning::
 
 	Clearing the replay protection list breaks the security mechanisms of the mesh node, making it susceptible to message replay attacks. This should never be performed in a real deployment.
 
+``mesh test health-srv add-fault <Fault ID>``
+---------------------------------------------
+
+	Register a new Health Server Fault for the Linux Foundation Company ID.
+
+	* ``Fault ID``: ID of the fault to register (``0x0001`` to ``0xFFFF``)
+
+
+``mesh test health-srv del-fault [Fault ID]``
+---------------------------------------------
+
+	Remove registered Health Server faults for the Linux Foundation Company ID.
+
+	* ``Fault ID``: If present, the given fault ID will be deleted. If omitted, all registered faults will be cleared.
 
 Provisioning
 ============
 
-``mesh pb-gatt <val: off, on>``
--------------------------------
+To allow a device to broadcast provisioning beacons, the :kconfig:option:`CONFIG_BT_MESH_PROV_DEVICE` configuration option must be enabled, along with either :kconfig:option:`CONFIG_BT_MESH_PB_GATT` or :kconfig:option:`CONFIG_BT_MESH_PB_ADV`.
```

Comment:
Can this description be moved under corresponding commands?  `CONFIG_BT_MESH_PB_GATT` under `mesh prov pb-gatt` and `CONFIG_BT_MESH_PB_ADV` under `mesh prov pb-adv` command.

Generally it would be good if a description for each command would tell which Kconfig option is needed for the command to show up in the shell. Or will it overload the documentation? What do you think?

Reply:
Yes, I think that is a good idea